javascript/javascript1/week1/pizza-exercise.js

```diff
@@ -0,0 +1,7 @@
+console.log("I Love Pizza");
+
+const myFavorite = 'Chicken pizza';
+const pizzaPrice = '70';
```

Comment:
Since this is a number and we might want to calculate with it in the future, you should declare it as a number:
`const pizzaPrince = 70;`